bin/templates/dbs/biogrid.sh

```diff
@@ -1,3 +1,7 @@
+# Getting the protein-protein interactions
 wget https://downloads.thebiogrid.org/Download/BioGRID/Latest-Release/BIOGRID-MV-Physical-LATEST.tab2.zip
 unzip BIOGRID-MV-Physical-LATEST.tab2.zip
-mv *.tab2.txt tab2
\ No newline at end of file
+mv *.tab2.txt tab2
+# if not at root of gwas-tools directory, change path so it gets into gwas-tools/test/data/
+mv tab2 test/data/
```

Comment:
What's the purpose of this line?

Reply:
It is to add the biogrid PPI table to the test/data and named as in the command line you provide in the README. We could also directly add the file into the test/data if that is authorized by the Biogrid license. Without it your example lines for network-guided GWAS in your README don't work.

Reply:
I see. If it's for testing purposes, I'd rather have a much shorter files containing a few made-up interactions. Hopefully one such that some "significant" results are output at the end in most methods.

Reply:
Then the whole biogrid.sh file should be removed I think